packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel.stories.jsx

```diff
@@ -45,149 +56,206 @@ export default {
   },
 };
 
-const renderUIShellHeader = (
-  open,
-  setOpen,
-  notificationTriggerRef,
-  hasUnreadNotifications
-) => (
-  <HeaderContainer
-    render={() => (
+const defaultProps = {
+  data: 0,
+  open: true,
+  onDoNotDisturbChange: action('Toggled "Do not disturb"'),
+  onViewAllClick: action('Clicked "View all"'),
+  onSettingsClick: action('Clicked gear icon'),
+};
+
+// Create a new notification.
+const newNotification = () => {
+  // Pick a random notification from the existing data set.
+  const random = Math.floor(Math.random() * data.length);
+  const notification = { ...data[random] };
+  // Update dynamic data.
+  notification.id = uuidv4();
+  notification.timestamp = new Date();
+  notification.unread = true;
+
+  return notification;
+};
+
+const Template = (args) => {
```

Comment:
`stories.jsx` has been refactored to more modern standards.

The **data** control now shows a dropdown with "Sample data" and "Empty data". We no longer need an "Empty" story.